docs/configuration.rst

```diff
@@ -472,3 +477,69 @@ Optional Settings
 All other settings for Connect like security, monitoring interceptors, producer and consumer overrides can be passed to the Docker images as environment variables. The names of these environment variables are derived by replacing ``.`` with ``_``, converting the resulting string to uppercase and prefixing it with ``CONNECT_``. For example, if you need to set ``ssl.key.password``, the environment variable name would be ``CONNECT_SSL_KEY_PASSWORD``.
 
 The image will then convert these environment variables to corresponding Connect config variables.
+
+-----------
+KSQL Server
+-----------
+
+The KSQL Server image can be configured through environment variables prefixed with KSQL_, uppercasing each word in the parameter name, and separating each word with an underscore (_) instead of periods (.). For example, bootstrap.servers becomes KSQL_BOOTSTRAP_SERVERS, and ksql.service.id becomes KSQL_KSQL_SERVICE_ID.
+
+.. sourcecode:: bash
+
+  docker run -d \
+    -p 127.0.0.1:8088:8088
+    -e KSQL_BOOTSTRAP_SERVERS=localhost:9092 \
+    -e KSQL_LISTENERS=http://0.0.0.0:8088 \
+    -e KSQL_KSQL_SERVICE_ID=ksql_test_service_id_ \
+    -e KSQL_KSQL_STREAMS_REPLICATION_FACTOR=3 \
+    -e KSQL_KSQL_SINK_REPLICAS=3 \
+    -e KSQL_KSQL_STREAMS_STATE_DIR=/ \
```

Comment:
Should this be `/docker/container/path` so that the state stores are run on the mounted volume?

Reply:
good catch. fixed.